packages/calcite-components/src/components/time-picker/time-picker.tsx

```diff
@@ -309,6 +312,11 @@ export class TimePicker
     }
   }
 
+  @Listen("mousedown")
+  hostMouseDownHandler(): void {
```

Comment:
For consistency, can you drop `host` from the name? Bonus points if this is moved to the `<Host>` element itself.

Reply:
It needs to use the `@Listen` decorator for it to work correctly, for some reason.  When I inline the pointerdown handler on the Host element, the `focusHandler` method is getting an out-of-date value for `pointerActivated`, resulting in the flash of focus styling.  I can go into more detail if you'd like to see it in action.

Reply:
Interesting. Both should behave the same. 🤔

In order to get this moving for the upcoming release, let's focus on:

* dropping `host` for consistency
* using `pointerdown` vs `mousedown`
* confirming if this is low-risk
* if time permits, adding an E2E to cover this – this one can be tackled immediately after the release if we need more time as long as we create an issue or PR for tracking purposes

Afterwards, let's look at the change in behavior from moving this listener to `Host`.

Reply:
- [ ] dropping `host` for consistency
- [ ] using `pointerdown` vs `mousedown`
- [ ] confirming if this is low-risk
- [ ] if time permits, adding an E2E to cover this

All done!  The e2e test is comprehensive, testing all up and down nudge buttons, including milliseconds.